zerver/lib/export.py

```diff
@@ -1974,9 +1975,10 @@ def export_realm_wrapper(
     print()
```

Comment:
@timabbott can we replace empty `print()` into `\n` in the start of the next print statement? both seem to give the same output.

Reply:
Yeah, that works.